src/teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './add'
+
+class Teacher < Person
+  def initialization(specialization)
+    @specialization = specialization
+  end
```

Comment:
- [ ] As I can see also here you missed to pass the super() because you inheirt from a class so also you should pass the parameters so please add it and pass base class parameters like above in student class 💪🏻🔥

Reply:
Done 💯